tests/TaskTestCase.php

```diff
@@ -17,13 +17,15 @@ public function runTask(array $args, ?string $cwd = null): Process
     {
         $coverage = $this->getTestResultObject()?->getCodeCoverage();
 
-        $bin = $_SERVER['CASTOR_BIN'] ?? __DIR__ . '/../bin/castor';
+        $castorBin = $_SERVER['CASTOR_BIN'] ?? __DIR__ . '/../bin/castor';
+        $phpBin = !isset($_SERVER['PHP_BIN']) || '' === $_SERVER['PHP_BIN'] ? \PHP_BINARY : $_SERVER['PHP_BIN'];
```

Comment:
PHP_BINARY can be stripped, since we run test only on unix platform and the script has a shebang. It'll simplify a lot the env var. IMHO we can keep only one